src/search1_rpc/service.py

```diff
@@ -11,22 +11,30 @@
     - es_client.search
     - search1_conversion.convert_result
 """
-import jsonrpcbase
-import time
 
-from src.es_client import search
+import time
+import os
+from jsonrpc11base import JSONRPCService
+from jsonrpc11base.service_description import ServiceDescription
+from src.es_client.query import search
 from src.search1_conversion import convert_params, convert_result
-from src.utils.config import config
 from src.utils.logger import logger
+from src.search1_rpc.errors import trap_error
+
+description = ServiceDescription(
```

Comment:
Is there a reason this is generic?

Reply:
No, that was just an oversight. I had copied an example service description from the jsonrpc11 library,  but not changed it!